packages/compass-data-modeling/src/components/drawer/diagram-editor-side-panel.spec.tsx

```diff
@@ -272,6 +272,32 @@ describe('DiagramEditorSidePanel', function () {
     });
   });
 
+  it('should delete a collection', async function () {
+    const result = renderDrawer();
+    result.plugin.store.dispatch(selectCollection('flights.countries'));
+
+    await waitFor(() => {
+      expect(screen.getByLabelText('Name')).to.have.value('countries');
+    });
+
+    userEvent.click(
+      screen.getByTestId('data-modeling-drawer-actions-delete-action')
```

Comment:
You can keep this, but I would really suggest to get accustomed to use getByRole / getByLabelText / similar methods for that. This is just generally a test that closer to user experience and also a bit more stable as LG team sometimes messes up the html element prop propagation (and I think speifically for testid had some plans to replace them eventually with their own convention)